Javadoc warnings

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behavior
<!--- Tell us what should happen -->
Javadocs should be generated without warnings.

## Actual Behavior
<!--- Tell us what happens instead -->

`mvn javadoc:javadoc -D maven.javadoc.failOnWarnings` produces:

```
2 warnings
[WARNING] Javadoc Warnings
[WARNING] /home/wschoenborn/Projects/tracer/tracer-core/src/main/java/org/zalando/tracer/Flow.java:29: warning: no description for @return
[WARNING] * @return
[WARNING] ^
[WARNING] /home/wschoenborn/Projects/tracer/tracer-core/src/main/java/org/zalando/tracer/Flow.java:30: warning: no description for @throws
[WARNING] * @throws IllegalStateException
[WARNING] ^
```

## Possible Fix
<!--- Not obligatory, but suggest a fix or reason for the bug -->

- Write proper javadoc
- enable `failOnWarnings` in javadoc plugin configuration

## Steps to Reproduce
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
See above

## Context
<!--- How has this bug affected you? What were you trying to accomplish? -->
Release a new version

--- a/tracer-core/src/main/java/org/zalando/tracer/Flow.java
+++ b/tracer-core/src/main/java/org/zalando/tracer/Flow.java
@@ -30,12 +30,7 @@ public interface Flow {
 
     void readFrom(UnaryOperator<String> reader);
 
-    /**
-     *
-     * @return
-     * @throws IllegalStateException
-     */
-    String currentId();
+    String currentId() throws IllegalStateException;
 
     void writeTo(BiConsumer<String, String> writer);
 

